workspaces/libnpmpublish/lib/publish.js

```diff
@@ -205,7 +205,7 @@ const ensureProvenanceGeneration = async (registry, spec, opts) => {
   if (opts.access !== 'public') {
     try {
       const res = await npmFetch
-        .json(`${registry}/-/package/${spec.escapedName}/visibility`, opts)
```

Comment:
this has been making a double path slash call, `//` because registry domain ends with `/` and the path has it too, this was unintuitive to keep when nocking properly.